mona_sdk/client.py

```diff
@@ -250,8 +262,26 @@ def __init__(
             override_host=override_app_server_host
         )
         self.filter_none_fields_on_export = filter_none_fields_on_export
-        self._default_sampling_rate = default_sampling_rate
+
+        # Data sampling.
+
+        self._sampling_config_name = sampling_config_name
         self._context_class_to_sampling_rate = context_class_to_sampling_rate or {}
+        self._default_sampling_rate = default_sampling_rate
+
+        if self._sampling_config_name:
+            sampling_config_list = self.get_sampling_factors()
+            try:
+                sampling_config = sampling_config_list[0]
+            except IndexError:
+                raise MonaInitializationException("Config name does not exist.")
```

Comment:
... or is empty...?

Reply:
nit - instead of try you can be more straightforward and slightly improved performance if you just check if sampling config list exists and isn't empty, in which case raise the same exception, otherwise just go on to do what you did without the try/catch clause...